modules/sdk-coin-sol/src/sol.ts

```
@@ -453,11 +453,17 @@ export class Sol extends BaseCoin {
 
     await tssUtils!.deleteSignatureShares(txRequestId);
     const recreated = await tssUtils!.getTxRequest(txRequestId);
+    let txHex = '';
+    if (recreated.unsignedTxs) {
+      txHex = recreated.unsignedTxs[0]?.serializedTxHex;
+    } else {
+      txHex = recreated.transactions[0].unsignedTx.serializedTxHex;
+    }
```

we should throw an error if `txHex` is empty after this block